### PR TITLE
docs(readmes): update content

### PR DESCRIPTION
**Pull request by jcfranco**
**Related Issue:** #1145

## Summary

Updates non-generated readme content:

* Dropped outdated/redundant entries
* Moved code examples into separate [usage files](https://stenciljs.com/docs/docs-json#usage).

**Note**: the readmes will be updated by the Stencil build in a separate PR.

### CODE REVIEW

**Review by driskull — APPROVED** [review]
awesome sauce! 🎉 We needed to do some of this readme to usage moving stuff :)

File: src/components/calcite-accordion/readme.md [hunk 1]
```diff
@@ -1,16 +1,6 @@
 # calcite-accordion
 
-Calcite-accordion can be used to present content in collapseable sections. Configuration options exist for visual style (theme, icon-position, scale, appearance), and selection functionality. Selection modes include "multi", "single", and "single-persist".
```
Comment (paulcpederson) [inline src/components/calcite-accordion/readme.md:3]: Is there a reason you dropped the description when you moved this over?

File: src/components/calcite-inline-editable/calcite-inline-editable.tsx [hunk 2]
```diff
@@ -14,6 +14,9 @@ import { getElementProp } from "../../utils/dom";
 import { Scale, Theme } from "../interfaces";
 import { TEXT } from "./resources";
 
+/**
```
Comment (paulcpederson) [inline src/components/calcite-inline-editable/calcite-inline-editable.tsx:17]: Oh nice, I didn't know you could doc slots like this, very cool

File: src/components/calcite-accordion/readme.md (hunk 1 above)
Comment (jcfranco) [reply src/components/calcite-accordion/readme.md:3]: I updated part of it, but it wasn't dropped.

**Review by paulcpederson — APPROVED** [review]

File: src/components/calcite-accordion/readme.md (hunk 1 above)
Comment (paulcpederson) [reply src/components/calcite-accordion/readme.md:3]: nvm, read the diff wrong

**Comment by paulcpederson** [comment]
Good to merge, IMO

**Pull request merged**